### DISCUSSION TITLE
Allies Soldier with Browning

### DISCUSSION THREAD

**Discussion by u1ps** [etlegacy/etlegacy#2030] [Ideas]
M2 Browning is 38kg, MG 42 is 11,6kg

You dont handle a Browning on a battlefield as 1 single soldier. From a logical perspective Single soldier running around with browning is unrealistic. I always found it silly seeing Soldier jumping around with browning, although it is a game after all. Just wanted to state a point that Soldier with MG is more logical as it is in vanilla. choice for browning is unclear for me but maybe someone can clarify why it is so? perhaps to create more variety between allies and axis team?

**Comment by wolfetplayer** [comment]
I think M1919 Browning(featured in ETL) is mistaken with M2 Browning here. It is possible to carry it just like mg42. (its 14kg)

**Comment by Aciz** [comment]
@wolfetplayer is right here, the weapon is referred as to having .30 caliber ammunition as well, which matches the M1919, M2 uses .50cal ammo.

![image](https://user-images.githubusercontent.com/14221121/171433191-43101af7-8544-47f4-944f-286027724155.png)

**Reply by u1ps** [reply]
yeah i also did some research and its M1919 for sure and it makes sense. Maybe just add the model number M1919 behind name in Limbo menu ;)
